irods/message/__init__.py

```diff
@@ -180,7 +180,7 @@ def ET(xml_type=(), server_version=None):
 
 logger = logging.getLogger(__name__)
 
-IRODS_VERSION = (4, 3, 4, "d")
+IRODS_VERSION = (5, 0, 1, "d")
```

Comment:
Don't forget to link this change to #743.

Reply:
ok.  I'm all right with leaving it in this PR, even tho' it is strictly unrelated.  That way we've a guarantee it will be included.